submissions/mxmgny/style/style.css

```diff
@@ -0,0 +1,70 @@
+aside{
+    width: 250px;
+    background-color:#f1f1f1;
+    position: absolute;
+    left: 0;
+    right: 0;
+    /* display: none; */
+    }
+button.active{
```

Comment:
Avoid styling by tag add custom styles on class.

Reply:
okay, thanks, I thought for this task it is acceptable.

Reply:
@mxmgny Styling by tag name is a bad practice in general. Also, you can read about [BEM](https://ru.bem.info/methodology/quick-start/). Using proper naming will prepare you for the changes in design of the website in future.

Reply:
Thank you for that article it was very interesting and useful. I implemented BEM approach in my code. Please review if you can